cdk_lib/dpdash-cdk-stack.test.ts

```diff
@@ -0,0 +1,135 @@
+import { Capture, Match, Template } from 'aws-cdk-lib/assertions';
+import * as cdk from 'aws-cdk-lib';
+import { DpdashCdkStack } from './dpdash-cdk-stack';
+
+describe('DPDashCDKStack', () => {
+  const OLD_ENV = process.env
+  const setEnv = (overrides = {}) => {
```

Comment:
my suggestion is to put this function in the fixtures file

Reply:
Good call